code/modules/mob/living/silicon/pai/examine.dm

```diff
@@ -1,2 +1,13 @@
-/mob/living/silicon/pai/examine(mob/user) //removed as it was pointless...moved to the pai-card instead.
-	return
\ No newline at end of file
+/mob/living/silicon/pai/examine(mob/user)
+	var/msg = "<span class='info'>*---------*\nThis is [bicon(src)] <EM>[src]</EM>!\n"
+	//expand on PAI status here
+	msg += "*---------*</span>"
+	to_chat(user, msg)
+	if(pai_law0 && isobserver(user) && !istype(user,/mob/dead/observer/deafmute))
+		var/mob/dead/observer/fag = user
```

Comment:
lad the github police are going to come for us some day, just use O or some normal var name

Reply:
i literally copied those parts from how its done with silicons tho

Reply:
You're right. We are going to be beheaded one of these days. One day without any warning, whole repo over gamer words.